app/generators/contestantRoundScoreGenerator.tsx

```diff
@@ -1,27 +1,22 @@
 import { ITableRowData } from "../utils/wikiFetch"
-import { getTeamList, ITeam } from "../utils/wikiQuery"
+import { ITeam } from "../utils/wikiQuery"
+import { getNumberOfRounds } from '../utils/teamListUtils';
 import Team from '../models/Team'
 import LeagueStanding from '../models/LeagueStanding'
 
 interface Dictionary<T> {
     [Key: string]: T;
 }
 
-async function generateContestantRoundScores(dataFetcher: () => Promise<ITableRowData[]>, listOfContestantLeagueData: any[]) {
-
-    const wikiContestants = await dataFetcher()
-    const pageData = getTeamList(wikiContestants)
-
+export async function generateContestantRoundScores(dataFetcher: () => Promise<ITableRowData[]>, getCompetitorList: any, listOfContestantLeagueData: any[]) {
+    const wikiData = await dataFetcher();
+    const pageData = getCompetitorList(wikiData);
```

Comment:
fft: I don't love this, mostly because I think there are situations where the "competitor" is actually a team... I would also be inclined to consider what it is doing more abstractly... I don't really have a specific suggestion at this time though 🤷 

lastly  what gets assigned back to the thing currently called `pageData` we should probably come up with a better name for that 🤔

Reply:
Agreed. I'll add notes in the code to think of a better name